package.json

```diff
@@ -11,18 +11,20 @@
     "chimp-test": "chimp chimp.config.js",
     "chimp-test:watch": "yarn run chimp-test --watch",
     "dev": "concurrently \"yarn run dev-client\" \"yarn run dev-server\" \"yarn run dev-parity\"",
-    "dev-parity": "yarn parity",
-    "dev-client": "yarn run clean && yarn run prep && NODE_ENV=test webpack --watch --colors --progress",
-    "dev-server": "NODE_ENV=test nodemon --watch src/server --watch node_modules src/server/index.js localhost:8080 ",
+    "dev-parity": "yarn run parity",
```

Comment:
Just as a note, I think we should always include `run` because if we unknowingly name a command the same as the name of a built-in `yarn` command, leaving out `run` will cause the built-in command to be run instead (we hit this issue w/ `yarn clean` before)